src/utils/helper-functions.tsx

```diff
@@ -109,3 +109,11 @@ export const spliceSx = (defaultStyles?: SxProps, newStyles?: SxProps) =>
     ...defaultStyles,
     ...newStyles
   } as SxProps)
+
+export const getNumberOfYears = function (
+  createdAt: string,
+  updatedAt: string
+) {
+  const difference: number = new Date(updatedAt) - new Date(createdAt)
+  return Math.round(difference / (1000 * 3600 * 24) / 365)
+}
```

Comment:
First of all, convert to an arrow function, then what will be displayed on the page when I am registered on this website for 20 days?
Nothing.
I guess you need to implement a function that will be returned days and years.

Reply:
Fixed.